docs/diff/formatters.md

```diff
@@ -0,0 +1,36 @@
+# Formatting the Diff Output
+
+The default output format for the diff tool is JSON. For a human-readable format, you can use `toFormattedString`.
```

Comment:
Suggest to "JSON is the default output format. Use `toFormattedStrong`' to view on the user console.